### ISSUE TITLE
Plumatic/schema?

### ISSUE THREAD

**Issue by theronic** [pupeno/free-form#29]
Did you consider [Plumatic's schema](https://github.com/plumatic/schema) to form the basis of form shapes?

**Comment by pupeno** [comment]
Not really. It's an interesting idea but it would require some extra annotations to make the all field types work and the output would be very strict. You wouldn't be able to perform customizations such as having two fields side by side because they are smaller (think first name and last name). 

Now, I can see the Vale if you are building a simple CRUD interface to lots of different models. I don't have the bandwidth to develop this but if this is something you want to do, I'm happy to participate in the design and make room for that in Free Form.
